src/test/resources/ConfigurationTest/project_cannot_use_buf_yaml_workspaces_and_the_protobuf_gradle_plugin_first/buf.work.yaml

```diff
@@ -0,0 +1,3 @@
+version: v1
```

Comment:
Oh, interesting. This is surprising that this works to have a v1 work file and v2 buf file in the same folder. I guess the constraint introduced in v1.34 was just if they are _both_ v1 files?

Do we have a test case covering when they are both v1 files? Instead of in the same directory, you'd have a `buf.work.yaml` at the root and then `buf.yaml` in the `workspace` folder.

Reply:
This is actually a failure case, I will clean it up.

Reply:
Ah, right.